src/coreclr/vm/jithelpers.cpp

```diff
@@ -2544,6 +2551,30 @@ static const BinderMethodID hlpDynamicToBinderMap[DYNAMIC_CORINFO_HELP_COUNT] =
 #include "jithelpers.h"
 };
 
+bool VMHELPDEF::IsDynamicHelper(size_t& dynamicFtnNum) const
```

Comment:
```suggestion
bool VMHELPDEF::IsDynamicHelper(DynamicCorInfoHelpFunc& dynamicFtnNum) const
```
Also, should this be `DynamicCorInfoHelpFunc*` to make it clear that this is out arg? It is unusual to return out args via `&` in this codebase.